### DISCUSSION TITLE
Light turn off with sound on?

### DISCUSSION THREAD

**Discussion by FuzzyMistborn** [dahlb/ha_hatch#26] [General]
Is it possible to have the light off but have the sound machine going?  Right now if I turn the light off it goes to 1% and still shows as "on".  Only way to turn the light off is to turn the sound off too which obviously isn't ideal as that would wake the kiddos.

**Comment by dahlb** [comment]
if you open an issue, specify your device type, and include you debug logs i expect we could get to the bottom of this for you.  I do just that daily having the light off with just sound on for my rest plus.

**Comment by dahlb** [comment]
v1.9.5 fixes the light on status to reflect when the rest plus is on and the light is off, this might have been the scenario you were experiencing
